source/features/batch-open-conversations.tsx

```diff
@@ -63,12 +63,49 @@ async function init(): Promise<void | false> {
 				Open selected
 			</button>
 		);
+	} else {
+		const issuesToolbar = select('#js-issues-toolbar')!;
+		issuesToolbar.prepend(
+			<div className="mr-3 d-none d-md-block">
+				<input type="checkbox" data-check-all="" aria-label="Select all issues" autoComplete="off"/>
+			</div>
+		);
+		issuesToolbar.append(
+			<div className="table-list-triage flex-auto js-issues-toolbar-triage">
+				<span className="text-gray table-list-header-toggle">
+					<span data-check-all-count="">1</span> selected
+					{' '}
+					<button type="button" className="btn-link rgh-batch-open-issues pl-3">Open selected</button>
+				</span>
+			</div>
+		);
+
+		const regExpPrNumber = /^issue_(\d*)$/;
+
+		const prRows = select.all('.js-issue-row');
+		for (const prRow of prRows) {
```

Comment:
None of these are "PRs", we've started calling them Conversations
```suggestion
		for (const conversation of select.all('.js-issue-row')) {
```